cmd/abapEnvironmentAssemblePackages.go

```diff
@@ -148,6 +148,9 @@ func starting(repos []abaputils.Repository, conn abapbuild.Connector) ([]buildWi
 			log.Entry().Infof("Packages %s is in status '%s'. No need to run the assembly", repo.PackageName, repo.Status)
 			buildsAlreadyReleased = append(buildsAlreadyReleased, buildRepo)
 		}
+
+		//as batch events in the ABAP Backend need a little time
+		time.Sleep(3 * time.Second)
```

Comment:
Did you consider that this could affect the runtime of unit tests?

Reply:
Actually no. As executing golang unit tests in VSC is so incredible slow I did not feel the extra 6s wait time... anyway solved with next commit